### ISSUE TITLE
Icons not visible with app created from Teams Toolkit

### ISSUE THREAD

**Issue by ekansh005** [OfficeDev/microsoft-teams-ui-component-library#115]
Hi,

I created an app for personal tab with Teams Toolkit. Now I am trying to use '@fluentui/react-teams' to render a List but the icons are not displayed in teams/standalone browser tab. As mentioned in issue #80 , I downgraded the versions but still the icons are not displayed. 

### Reproduction
The problem is reproduced here: https://codesandbox.io/s/fluentui-react-teams-issue-template-forked-0ms31h
My dependencies are:
```
  "dependencies": {
    "@fluentui/react-northstar": "^0.54.0",
    "@fluentui/react-teams": "^6.0.0",
    "@microsoft/teams-js": "^1.9.0",
    "@microsoft/teamsfx": "^0.5.1",
    "axios": "^0.21.1",
    "msteams-react-base-component": "^3.1.0",
    "react": "^16.8.0",
    "react-dom": "^16.8.0",
    "react-router-dom": "^5.1.2",
    "react-scripts": "^4.0.3"
  }
```

### Expected Behavior
The icons before 'Add', 'Filter' and 'row level actions' should have been displayed.

### Actual Behavior
![image](https://user-images.githubusercontent.com/439184/155880588-c1ee5167-8074-48cc-b81c-e1189aa07a95.png)

**Comment by thure** [comment]
This is a duplicate of #80.

Please see how it was resolved here: https://github.com/OfficeDev/microsoft-teams-ui-component-library/issues/80#issuecomment-856881488

**Comment by ekansh005** [comment]
@thure I had already checked that comment and applied the same in my sample. If you check the dependencies, those are exactly same as you mentioned in your solution. Still it doesn't work.

**Comment by thure** [comment]
Strange, looks like the SVGs have the same issue as in more recent versions of Northstar, though those didn't exhibit with correct dependencies in the past… It may be that `fluent-react-icons` further upstream is the issue.

If you'd like to submit a fix for #116, it would be welcome, I won't be able to address this very soon. The issue is SVG icons aren't getting an inherent width/height like they used to. I still think the underlying issue is equivalent so I'll keep this issue closed, so let's continue the conversation on #116.
